packages/drivers/odsp-driver/src/odspUtils.ts

```diff
@@ -123,7 +123,8 @@ export async function fetchHelper(
     }, (error) => {
         const online = isOnline();
         const errorText = `${error}`;
-
+        const spoRegex = /((http|https):\/\/.*\.com)/i;
+        const redactedErrorText = errorText.replace(spoRegex, "REDACTED (url)");
```

Comment:
I think a token like `REDACTED_URL` would probably be clearer when reading the redacted message (no space or parens)